afs/afs-mapdb-storage/src/main/java/com/powsybl/afs/mapdb/storage/NamedLinkSerializer.java

```diff
@@ -22,12 +22,14 @@
 
     @Override
     public void serialize(DataOutput2 out, NamedLink namedLink) throws IOException {
+        out.writeInt(MapDbStorageConstants.STORAGE_VERSION);
         UuidSerializer.INSTANCE.serialize(out, namedLink.getNodeUuid());
         out.writeUTF(namedLink.getName());
     }
 
     @Override
     public NamedLink deserialize(DataInput2 input, int available) throws IOException {
+        input.readInt(); // Storage version is retrieved here
```

Comment:
Maybe we could get the version and throw an exception if not version 0 to not forget to update the code the day we switch to version 1?

Reply:
I'm not sure this would ensure the code will be properly updated, since the first developer to move from version 0 to version 1 will correctly do the update (and remove exception throwing), but following developers using version 1 too may then not be aware they have to update their code. Proper unit tests may be sufficient to verify code update was properly made ?

Reply:
> 
> 
> I'm not sure this would ensure the code will be properly updated, since the first developer to move from version 0 to version 1 will correctly do the update (and remove exception throwing), but following developers using version 1 too may then not be aware they have to update their code. Proper unit tests may be sufficient to verify code update was properly made ?

Let's keep it as it is and see how to handle migration when we will need to. 
Could you update the migration guide to warm users that they have to destroy their db when upgrading?
https://github.com/powsybl/powsybl-core/wiki/Migration-guide-v3.1.0